app/src/main/java/swati4star/createpdf/fragment/ImageToPdfFragment.java

```diff
@@ -318,8 +318,9 @@ public void onRequestPermissionsResult(int requestCode,
     public void onActivityResult(int requestCode, int resultCode, Intent data) {
         super.onActivityResult(requestCode, resultCode, data);
         mButtonClicked = 0;
-        if (resultCode != Activity.RESULT_OK || data == null)
+        if (resultCode != Activity.RESULT_OK || data == null) {
             return;
+        }
```

Comment:
No need for the braces here. Don't touch unnecessary things please.

Reply:
removed the unnecessary things. Should I close this pull request and make another pull request?

Reply:
No, don't open another PR. Whatever the changes are, it should be done in this PR only.